lib/rules/require-spaces-around-generators.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Requires space before and after `*` in generator functions  ([named](#requirespacesinnamedfunctionexpression)
```

Comment:
Don't think you need to do the `([named](#requirespacesinnamedfunctionexpression)` part